packages/suite-web/e2e/tests/coinmarket/buy.test.ts

```diff
@@ -81,7 +81,7 @@ describe('Coinmarket buy', () => {
             const exchangeProvider: string[] = ['banxa', 'btcdirect'];
 
             // Get all displayed quotes
-            cy.get('[class*="CoinmarketOffersItem__OfferWrap"]')
+            cy.get('[class*="CoinmarketOffersItem__Offer"]')
```

Comment:
@andrcj Could you please refactor this test to use `data-testid`? 🙏  

It's a very dangerous to rely on these generated classnames. And also it restricts naming for our components.